lambda/otel/Dockerfile_3_2

```diff
@@ -10,7 +10,25 @@ WORKDIR /build/layer
 RUN bundle config set --local path 'ruby'
 RUN bundle update
 
+RUN for file in /build/layer/solarwinds_apm-*.gem; do \
```

Comment:
In my mind multiple local builds of `solarwinds_apm-*.gem` is unexpected for this CI/CD workflow, and we should bail and flag an error instead of trying to install each.

I'm also wondering if we can bypass the remove previous/install to different dir/copy over, by having `lambda/build.sh` comment out the `gem 'solarwinds_apm'...` line in Gemfile when installing from local, then here do something like `GEM_HOME=./ruby gem install $file`

Reply:
Thanks for the suggestion. I have updated them.